src/librustc/diagnostics.rs

```diff
@@ -368,6 +368,73 @@ loop. Without a loop to break out of or continue in, no sensible action can be
 taken.
 "##,
 
+E0282: r##"
+This error indicates that type inference did not result in one unique possible
+type, and extra information is required. In most cases this can be provided
+by adding a type annotation. Sometimes you need to specify a generic type
+parameter manually.
+
+A common example is the `collect` method on `Iterator`. It has a generic type
+parameter with a `FromIterator` bound, which is implemented by `Vec` and
+`VecDeque` among others. Consider the following snippet:
+
+```
+let x = (1_i32 .. 10).collect();
```

Comment:
If you're still inclined to use something like char, there is the option of starting with a string and iterating over its chars, like this:

```rust
    const HELLO: &'static str = "hello";
    let v: Vec<_> = HELLO.chars().collect();
    println!("v: {:?}", v);
```